include/sys/dsl_scan.h

```diff
@@ -107,7 +110,9 @@ typedef enum dsl_scan_flags {
 typedef struct dsl_scan {
 	struct dsl_pool *scn_dp;
 
-	boolean_t scn_pausing;
+	boolean_t scn_suspending;
+	uint8_t scn_scrub_state;
```

Comment:
should this be an enum?  this struct is in memory and there aren't many instances of it, so using a larger type is fine.